client/src/curriculum/overview.tsx

```diff
@@ -0,0 +1,41 @@
+import { HydrationData } from "../../../libs/types/hydration";
+import { CurriculumDoc, CurriculumData } from "../../../libs/types/curriculum";
+import { ModulesList } from "./modules-list";
+import { topic2css, useCurriculumDoc } from "./utils";
+import { PrevNext } from "./prev-next";
+import { RenderCurriculumBody } from "./body";
+import { CurriculumLayout } from "./layout";
+
+import "./index.scss";
+
+export function CurriculumModuleOverview(
+  props: HydrationData<any, CurriculumDoc>
+) {
+  const doc = useCurriculumDoc(props as CurriculumData);
+  const [coloredTitle, ...restTitle] = doc?.title?.split(" ") || [];
```

Comment:
Why do we highlight the first word in [**Getting** started modules](https://developer.allizom.org/en-US/curriculum/getting-started/)?

Don't we rather want to highlight everything except the last word ("modules")?